src/components/Checkbox/Checkbox.js

```diff
@@ -103,6 +90,22 @@ const inputStyles = ({ theme }) => css`
     border-width: 2px;
     border-color: ${theme.colors.p500};
   }
+
+  &:checked + label::after {
+    transform: translateY(-50%) scale(1, 1);
+    opacity: 1;
+  }
+
+  &:checked + label::before {
+    border-color: ${theme.colors.p500};
+  }
+`;
+
+const checkboxWrapperBaseStyles = ({ theme }) => css`
+  label: checkbox;
+  &:last-of-type {
+    margin-bottom: ${theme.spacings.mega};
```

Comment:
Need to double check if this is mega spacing or kilo spacing.

Reply:
Then do it 😃